Remotion/Web/Development.WebTesting.IntegrationTests/TestExecutionScreenshotRecorderTest.cs

```diff
@@ -286,12 +286,6 @@ private bool IsAlertDialogPresent (IWebDriver selenium)
       {
         return false;
       }
-      catch (UnhandledAlertException)

```

Comment:
Unfortunately, you did not provide any context as to why this could simply be removed. I'm guessing a Selenium update fixed this?

Reply:
@Stefan-Ilic Our Chromedriver update in [RM-7152](https://re-motion.atlassian.net/browse/RM-7152) fixed it. I don't know the exact version that did change the exception behavior.

Reply:
Hmm, it was between ChromeDriver v2.2.38 (previously used version) and 2.2.44. Is there any meaningful information in the ChromeDriver release notes?

Reply:
I found the following changes made in 2.44:
```
Resolved issue 2615: Incorrect 'alert open error' for window handle call [[Pri-2]]
Resolved issue 2597: Support new unhandledPromptBehavior modes [[Pri-3]]
```
But I did not see an apparent relation to our workaround ([UnhandledAlertException related issues](https://bugs.chromium.org/p/chromedriver/issues/list?colspec=ID%20Status%20Pri%20Owner%20Summary&q=unhandledalertexception&can=1)).

Reply:
Okay, we should add this information to the jira ticket. in theory, we could try and do a quick test of chromedriver v2.43 to see if this re-introduces the problem, only I have no idea how much effort that would take and what sort of problems we would see while trying out the older version of chrome driver.